doorstop/core/tree.py

```diff
@@ -215,6 +215,14 @@ def create_document(
 
         """
         prefix = Prefix(value)
+
+        # Check if a document with the same name already exists in the tree.
+        for d in self.documents:
+            if d.prefix == value:
+                raise DoorstopError(
+                    "The document name is already in use ({}).".format(d.doc_path)
```

Comment:
I don't think we need a new property for this, `document.path` should work.

Reply:
Fixed. I'm a bit new to git and github, so I might have done a commit too much, see below. But the corrections should be there now. Please tell me how to correct any git mistakes I might have done.